src/Bem.js

```diff
@@ -146,13 +113,66 @@ export default function({ preset, naming }) {
                     cls && entities.push(cls);
 
                     return entities.map(entity => typeof entity === 'string'?
-                        entity : str(entity)).join(' ');
+                        entity : stringifyEntity(entity)).join(' ');
```

Comment:
stringifyEntity -> Entity.stringify ?

Reply:
No, its stringify for CSS class.  `Entity.stringify` is a key.

Reply:
I'm going to rename it to `tokenize`